### ISSUE TITLE
Disbalance

### ISSUE THREAD

**Issue by PeakyB1** [xJon/The-1.12.2-Pack#899]
aluminum wires from galactic craft can convert the energy of all mods automotically, so there is no point in a mod for energy conversion. I think it needs to be fixed in configs

**Comment by xJon** [comment]
This conversion feature is only available for Galacticraft, not all mods, and the Galacticraft wire has a lot of limitations. I do admit it removes the point of a converter when using only Galacticraft alongside other mods, but there are still other cases when conversion is necessary.
In regards to changing the configs, I'm somewhat torn because I enjoy going with Galacticraft's easier defaults. I'll leave this open for any further discussion from the community.

**Comment by PeakyB1** [comment]
> This conversion feature is only available for Galacticraft, not all mods, and the Galacticraft wire has a lot of limitations. I do admit it removes the point of a converter when using only Galacticraft alongside other mods, but there are still other cases when conversion is necessary. In regards to changing the configs, I'm somewhat torn because I enjoy going with Galacticraft's easier defaults. I'll leave this open for any further discussion from the community.

but I was able to power the quarry from buildcraft using ic2 mfsu and a cable from galacticraft

**Comment by xJon** [comment]
Yes, it works with a lot of things. But I'm not sure if having an alternative is a balance problem or not, as it's quite hacky and doesn't support all machines (e.g. Industrial Foregoing).

**Comment by PeakyB1** [comment]
> Yes, it works with a lot of things. But I'm not sure if having an alternative is a balance problem or not, as it's quite hacky and doesn't support all machines (e.g. Industrial Foregoing).

I think we need a vote)

**Comment by xJon** [comment]
Feel free to start up a discussion with others on Discord! If a lot of others agree, I'll definitely consider it :)
